MainApp/lib/app/models/userStatistics.js

```diff
@@ -0,0 +1,24 @@
+import mongoose from 'mongoose';
+import validate from 'mongoose-validator';
+const Schema = mongoose.Schema;
+
+var numberValidator =[
+    validate({
+        validator: 'isNumeric',
```

Comment:
Apparently this has been deprecated. We should just use the mongoose validation. We can change this later.